src/opendr/perception/object_detection_2d/nanodet/nanodet_learner.py

```diff
@@ -483,15 +648,15 @@ def eval(self, dataset, verbose=True):
             num_sanity_val_steps=0,
             logger=self.logger,
         )
-        if verbose:
+        if self.logger:
             self.logger.info("Starting testing...")
         return trainer.test(self.task, val_dataloader, verbose=verbose)
 
     def infer(self, input, threshold=0.35, verbose=True):
         """
         Performs inference
         :param input: input can be an Image type image to perform inference
```

Comment:
Type is defined in :type
```suggestion
        :param input: input image to perform inference on
```